### DISCUSSION TITLE
Rank Modification Utilizing Plan's Activity Index

### DISCUSSION THREAD

**Discussion by Alexrs94** [plan-player-analytics/Plan#3819] [Q&A]
Curious if anyone has implemented a way to have luck perms move people up and down tracks so that their prefix represents their activity index. Basically what I'm looking to do. Put inactive/regular/active, ext as their prefix per Plan's Activity Index scoring. Not sure how to hook into the scoring or if possible.

**Comment by AuroraLS3** [comment]
Here's the previous discussion related to the topic https://github.com/plan-player-analytics/Plan/issues/3208
